js/src/pages/settings/index.js

```diff
@@ -23,6 +23,7 @@ import RebrandingTour from '~/components/tours/rebranding-tour';
 import SetupEnhancedConversions from './enhanced-conversions/setup-enhanced-conversions';
 import ExperienceRatingBanner from '~/components/experience-rating-banner';
 import './index.scss';
+import useGoogleMCAccount from '~/hooks/useGoogleMCAccount';
```

Comment:
To be consistent with the codebase, let's have the `index.scss` import last and if possible group all hooks import together.